### PR TITLE
Rule Update (Batch 2)

### PR DESCRIPTION
**Pull request by nasbench**
- Added 5 more PowerShell scripts for the rule "file_event_win_powershell_exploit_scripts.yml"
- Created a new rule for "certoc" lolbin to cover the "Download" option as described in the LOLBAS project
- Created a specific rule for the "IEExec" lolbin to cover the "Download" option as described in the LOLBAS Project
- Updated some rules to use "OriginalFileName" in addition to the "Image" selection
- Updated some rules to increase coverage.

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_certoc_download.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    selection_img:
+        - Image|endswith: '\certoc.exe'
+        - OriginalFileName: 'CertOC.exe'
+    selection_cli:
+        CommandLine|contains: '-GetCACAPS'
+    condition: all of selection*
+falsepositives:
+    - Unknown
+level: high
+
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_certoc_download.yml:22]:
```
tags:
    - attack.command_and_control
    - attack.t1105
```

File: rules/windows/process_creation/proc_creation_win_cmdkey_recon.yml [hunk 2]
```diff
@@ -7,20 +7,22 @@ references:
...
     - attack.t1003.005
 logsource:
     category: process_creation
     product: windows
 detection:
-    selection:
-        Image|endswith: '\cmdkey.exe'
+    selection_img:
+        - Image|endswith: '\cmdkey.exe'
+        - OriginalFileName: 'CertOC.exe'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_cmdkey_recon.yml:20]: `OriginalFileName: 'cmdkey.exe'`

**Review by frack113 — CHANGES_REQUESTED** [review]
Some check

File: rules/windows/process_creation/proc_creation_win_lolbas_configsecuritypolicy.yml [hunk 3]
```diff
@@ -9,12 +9,15 @@ tags:
...
+modified: 2022/05/16
 logsource:
     category: process_creation
     product: windows
 detection:
     lolbas:
-        CommandLine|contains: ConfigSecurityPolicy.exe
+        - CommandLine|contains: ConfigSecurityPolicy.exe
+        - Image|endswith: '\ConfigSecurityPolicy.exe'
+        - OriginalFileName: 'ConfigSecurityPolicy'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbas_configsecuritypolicy.yml:20]: `OriginalFileName: 'ConfigSecurityPolicy.exe'`

File: rules/windows/process_creation/proc_creation_win_lolbas_extrac32.yml [hunk 4]
```diff
@@ -9,14 +9,17 @@ tags:
     - attack.t1105
 author: frack113
 date: 2021/11/26
+modified: 2022/05/16
 logsource:
     category: process_creation
     product: windows
 detection:
     lolbas:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbas_extrac32.yml:17]: Here we look for "Extracts the source CAB file..."
```
 selection_lolbas:
        - CommandLine|contains: extrac32.exe
        - Image|endswith: '\extrac32.exe'
        - OriginalFileName: 'extrac32.exe'
 selection_archive:
       - CommandLine|contains: '.cab'
 selection_options:
       CommandLine|contains:
            - /C
            - /Y
            - ' \\'
condition: all of selection_*
```

File: rules/windows/process_creation/proc_creation_win_shadow_copies_deletion.yml [hunk 5]
```diff
@@ -23,28 +23,38 @@ logsource:
...
+        - OriginalFileName: 'WBADMIN.EXE'
+    selection2_cli:
         CommandLine|contains|all:
             - delete
             - catalog
             - quiet # will match -quiet or /quiet
-    selection3:
+    selection3_img:
         Image|endswith: '\vssadmin.exe'
+        OriginalFileName: 'VSSADMIN.EXE'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_shadow_copies_deletion.yml:51]:
```
    selection3_img:
        - Image|endswith: '\vssadmin.exe'
        - OriginalFileName: 'VSSADMIN.EXE'
````

**Review by frack113 — APPROVED** [review]

**Review by Neo23x0 — APPROVED** [review]

**Pull request merged**